moveit_core/CMakeLists.txt

```
@@ -230,6 +229,16 @@ add_subdirectory(collision_distance_field)
 add_subdirectory(kinematics_metrics)
 add_subdirectory(dynamics_solver)
 
+find_package(pybind11 2.4.3 QUIET)
```

```suggestion
find_package(pybind11 2.5.0 QUIET)
```
to avoid (known and unknown) regressions